### PR TITLE
Update spice_clone.py

### PR DESCRIPTION
**Pull request by Ascent817**

### CODE REVIEW

**Comment by Ascent817** [comment]
Michael you should give me contributor access

File: spice_clone.py [hunk 1]
```diff
@@ -62,6 +62,9 @@
     "lol",
     "bruj",
     "bruh moment",
+    "arm wrestle me bruh",
+    "you're actually retarded",
+    "Who asked"
```
Comment (VoidAny) [inline spice_clone.py]: There should be a comma at the end here

**Comment by VoidAny** [comment]
Varun, fix your pr and I'll merge it

**Comment by VoidAny** [comment]
Also, you should use a better name for your commit, like "add more possible responses" or something: "Update spice_clone.py" is pretty vague.

**Review by VoidAny — CHANGES_REQUESTED** [review]

File: spice_clone.py (hunk 1 above)
Comment (Ascent817) [reply spice_clone.py]: You have to approve the changes, I can't approve my own

Comment (Ascent817) [reply spice_clone.py]: It says that pr authors cannot approve changes

Comment (VoidAny) [reply spice_clone.py]: I don't approve your changes in this pr though

**Review by VoidAny — APPROVED** [review]

**Pull request merged**